baselines/polars/README.md

```diff
@@ -0,0 +1,19 @@
+# Setup
+
+1. You need to drag the data folder (e.g. `tables-scale_1`) into the `tpch` dir.
+
+2. you need to run the command `python3 prepare_files.py {1}` to process required tables.
```

Comment:
This command I think converts .tbl files to .parquet files. This script would be useful if we could move it to `scripts/` folder. This way it could be more generally (not just for polars) used to create a directory in every data directory with parquet files. For example: `$ROOT_DIR/resources/tpc-h/data/scale=1/partition=1/parquet/` can contain parquet files corresponding to the `.tbl` files at `$ROOT_DIR/resources/tpc-h/data/scale=1/partition=1/`

Reply:
Since there are multiple `scripts/`, just double check with you, I need to make a `.sh` file with this line of code and push it into `DeepOLA/scripts/`?